backend/forecastbox/products/rjsf/jsonSchema.py

```diff
@@ -0,0 +1,233 @@
+# (C) Copyright 2024- ECMWF.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+# A pydantic implementation of JSON Schema for React JSON Schema Form (RJSF).
+# This schema defines the structure of data as expected by RJSF based on the JSON Schema specification.
+# https://rjsf-team.github.io/react-jsonschema-form/docs/
+
+from typing import Any
+from typing import Dict
+from typing import List
```

Comment:
I suggest replacing `Dict` and `List` with `dict` and `list`, we are on high enough python